docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,184 @@
+	<div>
+		<p>
+			It is strongly recommended to upgrade older versions of IE to IE11. If you still need to support IE9, you can use CanvasRenderer
+			instead. Note that CanvasRenderer only supports a small subset of WebGLRenderer's features.
+		</p>
+		<p>
+			For Internet Explorer, the table below lists avaiable renderers.
+		</p>
+		<table>
+			<thead>
+				<tr>
+					<th>Renderer</th>
+					<th>IE Version</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>WebGLRenderer</td>
+					<td>IE11+</td>
+				</tr>
+				<tr>
+					<td>CanvasRenderer</td>
+					<td>IE9+</td>
+				</tr>
+				<tr>
+					<td>SVGRenderer</td>
+					<td>IE9+</td>
+				</tr>
+				<tr>
+					<td>CSS3DRenderer</td>
+					<td>IE10+</td>
+				</tr>
+				<tr>
+					<td>CSS2DRenderer</td>
+					<td>IE9+</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+
+	<h2>Part of ECMAScript Language Features or Web APIs Used in three.js</h2>
+	<div>
+		<p>
+			Here are some features used in three.js. Some of them may require additional polyfills.
+		</p>
+		<table>
+			<thead>
+				<tr>
+					<th>Feature</th>
+					<th>Use Scope</th>
+					<th>Modules</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Typed Arrays</td>
+					<td>Source</td>
+					<td>BufferAttribute, BufferGeometry, etc.</td>
+				</tr>
+				<tr>
+					<td>Web Audio API</td>
+					<td>Source</td>
+					<td>Audio, AudioContext, AudioListener, etc.</td>
+				</tr>
+				<tr>
+					<td>WebVR API</td>
+					<td>Source</td>
+					<td>WebVRManager, etc.</td>
+				</tr>
+				<tr>
+					<td>Blob</td>
+					<td>Source</td>
+					<td>FileLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>performance</td>
+					<td>Source</td>
+					<td>Clock, etc.</td>
+				</tr>
+				<tr>
+					<td>requestAnimationFrame</td>
+					<td>Source</td>
+					<td>WebGLRenderer, VideoTexture, etc.</td>
+				</tr>
+				<tr>
+					<td>Promise</td>
+					<td>Examples</td>
+					<td>GLTFLoader, GLTF2Loader, WEBVR, etc.</td>
+				</tr>
+				<tr>
+					<td>Fetch</td>
+					<td>Examples</td>
+					<td>ImageBitmapLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>Web Workers</td>
+					<td>Examples</td>
+					<td>WWOBJLoader2, etc.</td>
+				</tr>
+				<tr>
+					<td>File API</td>
+					<td>Examples</td>
+					<td>GLTFExporter, etc.</td>
+				</tr>
+				<tr>
+					<td>URL API</td>
+					<td>Examples</td>
+					<td>GLTFLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>TextEncoder & TextDecoder</td>
+					<td>Examples</td>
+					<td>FBXLoader, GLTFLoader, PCDLoaderetc, STLLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>Pointer Lock API</td>
+					<td>Examples</td>
+					<td>PointerLockControls, etc.</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+
+	<h2>Polyfills</h2>
+	<div>
+		<p>Just import polyfills base on your requirements. Taking IE9 as an example, you need to polyfill at least these features:</p>
+		<ul>
+			<li>Typed Arrays</li>
+			<li>Blob</li>
+			<li>performance</li>
+			<li>requestAnimationFrame</li>
```

Comment:
Actually you don't need to polyfill this - it's used in the WebGLRenderer (only for VR I think?), but you can't use that anyway with IE9.